packages/workspace/src/generators/npm-package/npm-package.ts

```diff
@@ -30,6 +30,7 @@ function addFiles(
   const packageJsonPath = join(projectRoot, 'package.json');
   writeJson(tree, packageJsonPath, {
     name: join(`@${npmScope}`, options.name),
+    version: '1.0.0',
```

Comment:
Let's start off with `0.0.0` like we do for our root `package.json`? I think it's more inline with what the initial code of a package would be published as :shrug: